tasks/Linux/selinux-support.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: "Install ansible selinux support library"
+  package:
+    name: libselinux-python
+    state: present
+  become: True
+
+- name: "Install ansible selinux configure libraries"
+  package:
+    name: "{{ selibs_item }}"
+    state: present
+  with_items:
+    - policycoreutils-python
+    - libsemanage-python
+  loop_control:
+    loop_var: selibs_item
+  become: True
+  when: ansible_selinux.status == "enabled"
+
+- name: "Enable connections to HTTP port"
+  seport: ports="{{ slave_linux_selinux_ports }}" proto=tcp setype=http_port_t state=present
```

Comment:
Please, **always use the map syntax**, regardless of how many pairs exist in the map. Like this:
```yaml
- name: "Enable connections to HTTP port"
  seport: 
    ports: "{{ slave_linux_selinux_ports }}"
    proto: tcp
    setype: http_port_t
    state: present
```

Reply:
agree, fixed